compiler/ballerina-parser/src/main/java/io/ballerinalang/compiler/internal/parser/tree/STNode.java

```diff
@@ -156,7 +156,7 @@ private void updateWidth(STNode[] children, int firstChildIndex, int lastChildIn
     private int getFirstChildIndex(STNode... children) {
         for (int index = 0; index < children.length; index++) {
             STNode child = children[index];
-            if (SyntaxUtils.isSTNodePresent(child) && child.widthWithMinutiae != 0) {
+            if (SyntaxUtils.isSTNodePresent(child) && child.widthWithLeadingMinutiae != 0) {
```

Comment:
Can you please create an issue explaining the bug here?

Reply:
This seems to be unnecessary now. I'll revert this.